.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: node_js
+node_js:
+- 10
+services:
+  - docker
+install: npm install
+
+stages:
+  - lint
+  - name: build docker image
+    if: branch != master || type != push
+  - name: deploy docker image
+    if: branch = master && type = push
+
+jobs:
+  include:
+    - stage: lint
+      script: npm run lint
```

Comment:
We can run 'npm run lint -- --fix' and it can also fix issues he knows how to fix,
what do you think?
Same for all the other builds

Reply:
It won't help since it just run there and does not commit to our repo.
It is possible to fix these errors automatically though, but it requires a bot, which I'm not going to build right now :laughing: